Tasks/AlexUlanov/Task1/rastsetVkladaV2.0.py

```diff
@@ -0,0 +1,40 @@
+def beSureIsDigit():
+    global x  
```

Comment:
it's possible to achieve the same result without global, try not to use except for those situations when there is no other choice